docker/docker-compose.yaml

```diff
@@ -24,6 +25,7 @@ services:
       - prompt_studio_data:/app/prompt-studio-data
       - ./workflow_data:/data
       - ${TOOL_REGISTRY_CONFIG_SRC_PATH}:/data/tool_registry_config
+      - ../backend/entrypoint.sh:/app/entrypoint.sh
```

Comment:
@chandrasekharan-zipstack Entrypoint is copied to base image layer itself, hence we don't need another volume mount here.
Please revert.